src/main.cpp

```diff
@@ -4720,7 +4732,7 @@ void CheckForFutileSync()
             {
                 mapAlreadyAskedFor.clear();
                 printf("\r\nClearing mapAlreadyAskedFor.\r\n");
```

Comment:
It does not actually clear `mapAlreadyAskedFor`.

Reply:
I don't understand, why wouldn't it clear `mapAlreadyAskedFor` ?